libcudacxx/test/libcudacxx/CMakeLists.txt

```diff
@@ -168,6 +168,7 @@ function(libcudacxx_add_test target_name_var source)
 
   add_executable(${target_name} "${source}")
   cccl_configure_target(${target_name} DIALECT ${CMAKE_CUDA_STANDARD})
+  target_include_directories(${target_name} PRIVATE "${libcudacxx_SOURCE_DIR}/test/libcudacxx/cuda/ccclrt/common")
```

Comment:
Might be out of scope for now, but we should eventually make this generic so we don't have to do this for every test that needs common headers. Something like a generic check if a `common` dir exists next to the source file and include it if so.